### ISSUE TITLE
Update to new mockup

### ISSUE THREAD

**Issue by jelly** [cockpit-project/cockpit-files#452]
That's an extremely old mockup at the top. We've had it on the left for several months.

Here's what files is supposed to (eventually) look like:

![image](https://github.com/cockpit-project/cockpit-files/assets/10246/ffe10fa4-a58d-4efb-a180-6d680885567a)

Basically: 
- The stuff on the left affects the stuff on the right. 
- The stuff on the top affects the stuff below it.

(This is basically how UI flow works overall, not just for Files.)

To break it down further, for Files:

1. Bookmarks affect location. 
2. Editing affects location. 
3. Then location. 
4. Then filtering and sorting changes the look of the location. 
5. Then actions for that location (which is affected by everything but before, _except view filtering and sorting_, actually).

So 1 - 3 are inputs to your location (with 3 being your current status) and 4 - 5 how you see and what you do when there.

The widgets in the mockup are all PF conventions, including the inline edit (a standard input, with the buttons without frames, and the check first with the x after).

_Originally posted by @garrett in https://github.com/cockpit-project/cockpit-files/issues/422#issuecomment-2095567330_

**Comment by jelly** [comment]
Blocked by:  https://github.com/cockpit-project/cockpit-files/pull/384

**Comment by allisonkarlitskaya** [comment]
fwiw, I like having separate functional groupings for:

 - everything to do with navigation (bookmarks, breadcrumb, edit location, etc.)
 - everything to do with how the current location is shown (sort, filter, grid/list, show-hidden, etc.)

and I like that the second group is located physically closer to the area that contains the items (ie: inside of the big white box) instead of at the top beside the navigation area.  I also feel like this is less likely to run out of space on mobile.

**Comment by garrett** [comment]
1. Latest version of all mockups are now at https://github.com/cockpit-project/cockpit-files/discussions/471
2. A WIP discussion about merged vs. unmerged bars, with mockups, is now at https://github.com/cockpit-project/cockpit-files/discussions/474

We should try to keep the exploratory and discussion related discussions as completely separate topics, as to not clutter the design doc. (We can make lots of posts in discussions and close or abandon ones that are no longer relevant, but threads within a discussion get gnarly pretty quickly.)
